app/routes/circulars/circulars.server.ts

```diff
@@ -272,6 +272,37 @@ export async function get(
   return result as Circular
 }
 
+/** Check if a Circular exists by ID. */
+export async function exists(
+  circularId: number,
+  version?: number
+): Promise<boolean> {
+  const circularVersions = await getDynamoDBVersionAutoIncrement(circularId)
+  const result = await circularVersions.get(version)
```

Comment:
- Directly query the circulars table, since version numbers are irrelevant here.
- Use a ProjectionExpression that limits to the returned data to the bare minimum.